lib/project.js

```diff
@@ -139,11 +139,8 @@ function localVersions(metadata, callback) {
     .pipe(util.vinyl.semver())
     .pipe(concat({ encoding: 'object'}, function (versions) {
       versions = _(versions)
-        .map(function (v) {
-          return v && cleanVersion(v);
-        })
         .filter(function (v) {
-          return !!v;
+          return !!v && !!cleanVersion(v);
```

Comment:
you don't really need to coerce to boolean. Returning `v && cleanVersion(v)` would work too as filter only cares about whether the returned value is truthy or falsy